assets/app/scripts/filters/util.js

```diff
@@ -117,6 +117,9 @@ angular.module('openshiftConsole')
           }
           unit += (amount === "1" ? "core" : "cores");
           break;
+        case "storage":
+          unit = unit.substring(0, 1);
```

Comment:
dont drop the i in Gi, it actually does mean something different :)

On Tue, Dec 22, 2015 at 3:11 PM, Fabiano Franz <notifications@github.com>
wrote:

> In assets/app/scripts/filters/util.js
> <https://github.com/openshift/origin/pull/6358#discussion_r48292989>:
>
> > @@ -117,6 +117,9 @@ angular.module('openshiftConsole')
> >            }
> >            unit += (amount === "1" ? "core" : "cores");
> >            break;
> > +        case "storage":
> > +          unit = unit.substring(0, 1);
>
> Sometimes it's just G and sometimes it's Gi as referred by some docs
> <https://docs.openshift.org/latest/dev_guide/persistent_volumes.html#requesting-storage>.
> I'll keep substring-ing it but I'll add B like memory does, so that we
> have GB.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6358/files#r48292989>.
>